src/components/MessagesList/MessagesList.vue

```diff
@@ -827,6 +987,10 @@ export default {
 			this.debounceUpdateReadMarkerPosition()
 		},
 
+		endScroll() {
+			this.isScrolling = false
```

Comment:
Doesn't work the last time, after list is loaded and rendered (scrolling start might be caused by reference widgets)
Just to note, don't know how we could fix it, and it remains until user scrolls, so you have enough time to see the last messages date => fine by me

Reply:
Fixed